src/reducers/MealReducer.js

```diff
@@ -0,0 +1,21 @@
+import { mealActionTypes as actionTypes } from '../constants/actionTypes';
+import { createReducer } from '../utils';
+
+const initialState = {
```

Comment:
this state machine is not very meaningful

Reply:
There’s nothing meaningful to put there other than to see if we’re waiting for a request